### PR TITLE
Resolved linter semicolon issues. 

### LINKED ISSUE #48: Clean up @typescript-eslint/no-extra-semi

### PR DESCRIPTION
**Pull request by parthpandey1**
Changed rule in .eslintrc.cjs  for extra semicolons to show error instead of warning. Resolved extra semicolon issues.

### CODE REVIEW

File: .eslintrc.cjs [hunk 1]
```diff
@@ -5,7 +5,7 @@ module.exports = {
     root: true,
     rules: {
         "@typescript-eslint/no-var-requires": "warn",
-        "@typescript-eslint/no-extra-semi": "warn",
+        "@typescript-eslint/no-extra-semi": "error",
```
Comment (Jooms) [inline .eslintrc.cjs:8]: Just remove the entire line. `warn` was an override. Setting it to `error` is letting it go back to the default.

**Review by Jooms — APPROVED** [review]
Approved with 1 comment to fix.

**Review by jayrevolinskyjr — CHANGES_REQUESTED** [review]
I agree with @Jooms , removing the no-extra-semi line will resolve this

**Review by jayrevolinskyjr — DISMISSED** [review]
LGTM

**Comment by decoles** [comment]
Looking good!

**Review by decoles — APPROVED** [review]

**Comment by parthpandey1** [comment]
rerun all

**Comment by parthpandey1** [comment]
retest all

**Review by hardikpatil — APPROVED** [review]

**Review by jayrevolinskyjr — APPROVED** [review]

**Pull request merged**